.github/workflows/docs.yml

```diff
@@ -19,8 +19,7 @@ jobs:
         run: |
           git config --local user.email "infra@linea.ai"
           git config --local user.name "Humble bot servant"
-          git add .
-          git commit -m "Add tutorials to docs"
+          git diff --quiet && git diff --staged --quiet || git commit -am "Add tutorials to docs"
```

Comment:
What is the need for the `git diff`s here?

Reply:
discussed offline: git diff compares with the index (staged changes) and git diff --staged compares staged changes with the HEAD (remote). git add . is still needed because untracked changes are not caught by either.